stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,433 @@
+import '../../dist/rating-indicator.css';
+import './custom.scss';
+
+export default {
+    title: 'Components/Rating Indicator',
+    parameters: {
+        description: `
+The rating indicator can be used to rate content or to indicate.
+
+##Usage
+Use the rating indicator in forms, tables, or in a dialog box.
+        `
+    },
+    tags: ['f3', 'a11y', 'theme']
+};
+
+export const customIconSizes = () => `
+    <div class="example-container">
+        <span>Default:</span>
+        <div class="fd-rating-indicator fd-rating-indicator--hide-dynamic-text">
+            <div class="fd-rating-indicator__container" aria-label="Rating of this item is 2 out of 5">
```

Comment:
The label for the group should be more along the lines of describing what is being rated (ie. 'Star Rating for X (out of y)' or just 'Star Rating (out of x)' if the context is going to be clear. The value for the item is already listed in text below.

Reply:
changed to `Star Rating (out of x)`